packages/redux-store/src/subscriptions/Billing/Information/saga.ts

```diff
@@ -47,13 +47,13 @@ function* loadPlan(planId: string) {
         id: planResponse.id,
         name: planResponse.name,
         description: planResponse.description,
-        price: planResponse.price.amount / 100,
-        currency: planResponse.price.currency,
+        price: (((planResponse.price && planResponse.price.amount) || 0) / 100).toFixed(2),
```

Comment:
add plus at the first to cast it from string to number
+(((planResponse.price && planResponse.price.amount) || 0) / 100).toFixed(2)